corehq/apps/reports/templates/reports/standard/base_template.html

```diff
@@ -86,6 +89,14 @@
 
 {% block page_content %}
     {% initial_page_data 'js_options' report.js_options %}
+
+    {# Needed for case reassignment interface #}
```

Comment:
I don't really love having initial page data in such a generic template that's specific to a single report; might make more sense to do these as data attributes on `data-interfaces-reassign-cases`. On the other hand, there's already a bunch of report-specific scripts on this page.